<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ì‚¼ì‚¼ì— íˆ¬ ë§¤ë¬¼ ì •ë³´ìˆ˜ì§‘ í”„ë¡œê·¸ë¨</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>551</width>
     <height>63</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NanumGothic</family>
     <pointsize>22</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;ì‚¼ì‚¼ì— íˆ¬ ë§¤ë¬¼ì •ë³´ í¬ë¡¤ë§ í”„ë¡œê·¸ë¨&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="margin">
    <number>15</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>160</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NanumGothic</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>í‚¤ì›Œë“œ</string>
   </property>
   <property name="margin">
    <number>5</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="keyword">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>451</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NanumGothic</family>
    </font>
   </property>
   <property name="placeholderText">
    <string>ì§€ì—­ í‚¤ì›Œë“œë¥¼ ì…ë ¥í•´ì£¼ì„¸ìš”. (ì˜ˆ : ì‚¬ë‹¹,ê°•ë‚¨)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="keyword_btn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>590</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NanumGothic</family>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ğŸ“Ší‚¤ì›Œë“œ ì„¤ì •í•˜ê¸°</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>511</width>
     <height>301</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NanumGothic</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid black;
border-radius: 5px;
padding: 8px;</string>
   </property>
   <property name="placeholderText">
    <string>ì—¬ê¸°ì—ì„œ ì§„í–‰í˜„í™©ì„ ë³´ì‹¤ ìˆ˜ ìˆìŠµë‹ˆë‹¤ :-)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="start_btn">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>590</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NanumGothic</family>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ğŸ” ì¶”ì¶œì‹œì‘</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reset_btn">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>590</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NanumGothic</family>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ğŸš€ ì´ˆê¸°í™”</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quit_btn">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>590</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NanumGothic</family>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>âŒ ì¢…ë£Œ</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NanumGothic</family>
     <pointsize>10</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>ì œì‘ : í¬ëª½ - ì˜ë…¸ëŠ”ì²«ëˆˆâ„ï¸</string>
   </property>
   <property name="margin">
    <number>5</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>521</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NanumGothic</family>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>âš ï¸ í‚¤ì›Œë“œë¥¼ í•œê°œë§Œ ë„£ìœ¼ì…”ë„ ë˜ê³ , ì½¤ë§ˆ(,)ë¡œ ë„ì–´ì“°ê¸° ì—†ì´ ì—°ê²°í•˜ì—¬ ì“°ì…”ë„ ë©ë‹ˆë‹¤! âš ï¸</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>521</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NanumGothic</family>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>âš ï¸ í‚¤ì›Œë“œê°€ ì—¬ëŸ¬ê°œì¸ ê²½ìš°, í•œ í‚¤ì›Œë“œì˜ ì‘ì—…ì´ ëë‚œ í›„ ë‹¤ë¥¸ í‚¤ì›Œë“œì— ëŒ€í•´ ì§„í–‰ë©ë‹ˆë‹¤!âš ï¸</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NanumGothic</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ë§Œë£Œ ë‚ ì§œ</string>
   </property>
  </widget>
  <widget class="QLabel" name="expire_date">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>441</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NanumGothic</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
